test/Vault.spec.ts

```diff
@@ -58,6 +58,7 @@ describe("Vault", () => {
 
   const DEFAULT_ADMIN_ROLE = constants.HashZero;
   const INVESTOR_ROLE = utils.keccak256(utils.toUtf8Bytes("INVESTOR_ROLE"));
+  const SETTINGS_ROLE = utils.keccak256(utils.toUtf8Bytes("SETTINGS_ROLE"));
```

Comment:
Out of curiosity is there a reason why we're not using the ADMIN_ROLE for this?

Is it because ADMIN_ROLE not defined in the contract so effectively it's the same thing?

Reply:
as a general practice, I would advise against relying on the default role for anything other than granting/revoking roles from other accounts

the admin role is the one that can grant "superuser" rights to everyone else. but if it's needed for things such as "setInvestPct", that means your backend, or whoever else needs to call that function, needs to be granted admin superuser permissions by extension.

it's best to have specific roles for specific things, even if they end up being the same account in the future